content/source/docs/enterprise/private/aws-setup-guide.html.md

```diff
@@ -137,6 +137,14 @@ clients and the PTFE application server. The certificate can be
 specified during the UI-based installation or the path to the
 certificate codified during an unattended installation.
 
+If a Classic or Application Load Balancer is used, SSL/TLS will be terminated there.
+In this configuration, the PTFE instances should still be configured to listen
+for incoming SSL/TLS connections but can do so using a self-signed certificate.
+
+The load balancer, being configured to send traffic to port 443 on the instances
```

Comment:
I might write this as "Since the load balancer is configured". Otherwise, looks good.